pkg/apis/alerting/v1/validate.go

```diff
@@ -18,6 +19,22 @@ func validComparionOperator(op string) error {
 	return nil
 }
 
+func (s *SyncerConfig) Validate() error {
+	if s.GatewayJoinAddress == "" {
+		return validation.Error("GatewayJoinAddress must be set")
```

Comment:
missing field errors should all be of the form
`fmt.Errorf("%w: %s", validation.ErrMissingRequiredField, <field name>)`

Reply:
Done